tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
no `strict`? :)

Reply:
I normally only use it for new code, as I don't want to change too much in the existing code